src/main/java/com/std/tothebook/api/controller/api/MyBookController.java

```diff
@@ -0,0 +1,46 @@
+package com.std.tothebook.api.controller.api;
+
+import com.std.tothebook.api.domain.dto.FindMyBookResponse;
+import com.std.tothebook.api.service.MyBookService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Tag(name = "독서기록")
+@RequestMapping("/api/myBook")
+@Controller
+@RequiredArgsConstructor
+public class MyBookController {
+
+    private final MyBookService myBookService;
+
+    @Operation(summary = "독서기록 리스트 조회")
+    @GetMapping("/")
+    public String getMyBooks(Model model){
+        long userId = 1;
```

Comment:
```suggestion
        long userId = 1L;
```
현재 문제가 되지 않고 크게 잘못된 것은 아니지만 long 타입은 값 세팅 시 뒤에 L을 붙여주시는 걸 추천합니다.
해당 변수에 값을 할당하기 위해 1이라는 값을 잠시 메모리에 저장해두게 되는데,
이때 L을 명시해주지 않으면 int형으로 설정되게 됩니다.
추후에 값을 넣어줄 때 int형보다 큰 숫자를 넣게 된 다면 오류가 발생합니다.
해서 L을 붙여주는 습관을 들여주시면 좋을 듯 합니다~

Reply:
회사에서 long 타입을 기본으로 사용하면서 L을 사용하지 않는 것이 손에 익었네요. 😅말씀해주신 것처럼 습관 들이도록 노력하겠습니다.